src/pal/tools/gen-buildsys-clang.sh

```diff
@@ -22,10 +22,16 @@ if which "clang-$2.$3" > /dev/null 2>&1
     then
         export CC="$(which clang-$2.$3)"
         export CXX="$(which clang++-$2.$3)"
+        if [ `uname` = "FreeBSD" ]; then
+            export LLVM_HOME=${LLVM_HOME:-/usr/local/llvm$2$3}
+        fi
 elif which "clang$2$3" > /dev/null 2>&1
     then
         export CC="$(which clang$2$3)"
         export CXX="$(which clang++$2$3)"
+        if [ `uname` = "FreeBSD" ]; then
+            export LLVM_HOME=${LLVM_HOME:-/usr/local/llvm$2$3}
```

Comment:
The problem is that this makes FreeBSD different from the other platforms. And we are actually trying to scan /usr/local/llvm-x.x in the CMakeLists.txt for the lldbplugin, starting from 3.8 and going down to 3.5 and try to use the highest version installed. Independent of what your clang version is, they don't need to match. And this happens even if you don't have LLVM_HOME set. Setting it is just a way to override the decision made in there.
So I'd recommend removing this change.